src/main/java/mate/academy/spring/model/dto/request/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.request;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private User user;
```

Comment:
why user in shopping cart response dto?

Reply:
I don't know, I actually added all fields from ShoppingCart model to dto.